plugins/widget/plugin.js

```diff
@@ -2901,6 +3004,80 @@
 		} );
 	}
 
+	function handleStylesOnDowncast( widget, widgetElement ) {
```

Comment:
Where are classes handled? I'm wondering if it won't be better to put them also in the same place as other methods of styling.

Reply:
Classes don't need special treatment here. They are already on element just like other attributes.